downstream/modules/hub/proc-create-synclist.adoc

```diff
@@ -30,7 +30,7 @@ The following domain names must be in the allow list:
 // ddacosta I don't know if a change will be needed here for Gateway as this is referring to the Console version of Hub. Will console pull in nav changes? Also, there is no repositories selection on the console version right now. 
 . Log in to `{Console}`.
 . Navigate to menu:Automation Hub[Collections].
-. Set the toggle switch on each collection to exclude or include it on your synclist.
-. To initiate the remote repository synchronization, navigate to {HubName} and select {MenuACAdminRepositories}.
-. Click the btn:[More Actions] icon *{MoreActionsIcon}* and select *Sync* to initiate the remote repository synchronization to your {PrivateHubName}.
+. Set the *Sync* toggle switch on each collection to exclude or include it on your synclist.
+. To initiate the remote repository synchronization, navigate to your {PlatformNameShort} and select {MenuACAdminRepositories}.
+. Click the btn:[More Actions] icon *{MoreActionsIcon}* and select *Sync* to initiate the remote repository synchronization to your {PrivateHubName}. //unclear which repo
```

Comment:
This would be any of the repositories the user chooses.

Maybe add an additional step:
. Select the repository or multiple repositories from the *Repositories* list view.
. Click the btn:[More Actions] icon and select *Sync* ...